extensions/core/interfaces/map/map.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * This mixin has common imports, methods & variables required to display map on 'input' or 'display'
+ */
+import mixin from "../../../mixins/interface";
+import leaflet from "leaflet";
+import "leaflet/dist/leaflet.css";
+
+const mapMixin = {
+  mixins: [mixin],
+  data() {
+    return {
+      map: null,
+      marker: null,
+      latlng: {
+        lat: null,
+        lng: null
+      }
+    };
+  },
+  watch: {
+    //TODO: Check if this watcher is required
+    //? Do we need to re-render map when theme changes from interface debugger?
```

Comment:
While there isn't an interface that sets other interfaces' options yet, there might be in the future.